Refresher/UI/PSPSetupForm.cs

```diff
@@ -26,18 +26,38 @@ public PSPSetupForm() : base("PSP Setup")
         
         foreach (DriveInfo drive in drives)
         {
+            bool isVita = false;
+            string pspEmuFolderName = "";
+
             try
             {
                 Console.WriteLine($"Checking drive {drive.Name}...");
-                
-                //If theres no PSP folder,
-                if (!Directory.Exists(Path.Combine(drive.RootDirectory.FullName, "PSP")))
+
+                // Match for all directories called PSP and PSPEMU
+                // NOTE: we do this because the PSP filesystem is case insensitive, and the .NET STL is case sensitive on linux
+                List<string> possiblePspMatches = Directory.EnumerateDirectories(drive.RootDirectory.FullName, "PSP", new EnumerationOptions
+                {
+                    MatchCasing = MatchCasing.CaseInsensitive,
+                    AttributesToSkip = 0
```

Comment:
Not that it matters, but is there a reason for this? Are the PSP/PSPEMU folders hidden?

Reply:
Yes, they're marked as both hidden and protected operating system files under Windows so we need this